src/svgp.jl

```diff
@@ -0,0 +1,68 @@
+struct SVGP end
+
+raw"""
+    approx_posterior(::SVGP, fz::FiniteGP, q::MvNormal)
+
+Compute the approximate posterior [1] over the process `f = fz.f`, given inducing
+inputs `z = fz.x` and a variational distribution over inducing points `q(u)` where `u =
+f(z)`. The approximate posterior at test points ``x^*`` where ``f^* = f(x^*)``
+is then given by:
+
+```math
+q(f^*) = \int p(f | u) q(u) du
+```
+which can be found in closed form.
+
+[1] - Hensman, James, Alexander Matthews, and Zoubin Ghahramani. "Scalable
+variational Gaussian process classification." Artificial Intelligence and
+Statistics. PMLR, 2015.
+"""
+function AbstractGPs.approx_posterior(::SVGP, fz::FiniteGP, q::AbstractMvNormal)
+    m, A = mean(q), cholesky(cov(q))
+    Kuu = cholesky(Symmetric(cov(fz)))
```

Comment:
does the cov(q) need a `Symmetric` as well? what's the difference to using AbstractGPs's `_symmetric` (that you also import)?

Reply:
I removed the import of `_symmetric` because I don't think it's really necessary here - and I suppose it's better not to depend on internals of AbstractGPs.

I suppose it should really be `cholesky(Symmetric(cov(q)))`, but that would break the zero overhead cholesky for `MvNormal` from [here](https://github.com/JuliaStats/Distributions.jl/pull/1373) no?

Reply:
I've changed it to dispatch on the type of `q` - which should solve this